applications/tari_console_wallet/src/ui/components/notification_tab.rs

```diff
@@ -4,9 +4,6 @@
 // are cleared.
 // Currently notifications are only added from the wallet_event_monitor which has
 // add_notification method.
-// TODO: auto delete old notifications. #LOGGED
-// TODO: add interaction with the notifications, e.g. if I have a pending transaction
```

Comment:
I don't think this TODO is worthwhile. All the events come from a single pane so there is only one place to go which is one right arrow press away. There is also no mouse interactivity in the console wallet so the concept of "clicking" doesn't exist.